test/destination/selector/test.css

```diff
@@ -0,0 +1,3 @@
+div#qunit {
+    position: relative;
+}
```

Comment:
fixed, well i took inspiration from here https://github.com/oat-sa/tao-core-ui-fe/blob/e3bff77a61f9cac5345acf49c269434c42ec0b30/test/contextualPopup/test.css

Reply:
Ok got it. But it was incomplete because the CSS also need to be loaded, like here: https://github.com/oat-sa/tao-core-ui-fe/blob/e3bff77a61f9cac5345acf49c269434c42ec0b30/test/contextualPopup/test.js#L1